scripts/gulp/plugins/gulp-component-menu-behaviors.ts

```diff
@@ -59,12 +74,32 @@ export default () => {
         variation.specification = getTextFromCommentToken(commentTokens, 'specification');
       }
 
-      result.push({
-        displayName: behaviorName,
-        type: componentType,
-        variations: variation,
-      });
-      cb();
+      // generate behavior description from 'behavior definition' file if no was found in behavior file
+      if (!variation.description && !variation.specification) {
+        const variationName = variation.name.replace('.ts', '');
+        const behaviorDefinitionName = `${variationName}Definition`;
+        const fluentUiDir = getPathDirByName(file.path, 'fluentui');
+
+        importFile(fluentUiDir, behaviorName, behaviorDefinitionName).then(file => {
+          const descriptionFromDefinition = file[behaviorDefinitionName].map(definition => {
+            return definition.stringify();
+          });
+          variation.description = descriptionFromDefinition.join('\r\n');
+          result.push({
+            displayName: behaviorName,
+            type: componentType,
+            variations: variation,
+          });
+          cb();
+        });
```

Comment:
I simplified this part, please check that it still works

Reply:
thank you very much, verified, tested, works as expected